language/tutorial/tutorial.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START full_tutorial_script]
+# [START import_libraries]
+import argparse
+
+from googleapiclient import discovery
+from oauth2client.client import GoogleCredentials
+# [END import_libraries]
+
+
+def authenticate():
```

Comment:
Done. I ended up removing the separate function, it may actually make sense to flatten the whole sample into a single helper that accepts a filename.